### PR TITLE
caido 0.41.0 (new cask)

### PR DESCRIPTION
**Pull request by i0ntempest**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/c/caido.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+cask "caido" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "0.41.0"
+  sha256 arm:   "eb2e9b61635231bab9f2829d4df70db00a5071c0469b08db1a3dd9e007d09737",
+         intel: "a57f6aa21cff95853697f20e29d2dfbbedb56bacd14ba177930b17ca92a6e093"
+
+  url "https://caido.download/releases/v#{version}/caido-desktop-v#{version}-mac-#{arch}.dmg",
+      verified: "caido.download/"
+  name "caido"
```
Comment (khipp) [inline Casks/c/caido.rb:10]:
```suggestion
  name "Caido"
```

File: Casks/c/caido.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  desc "Lightweight web security auditing toolkit"
```
Comment (khipp) [inline Casks/c/caido.rb:11]:
```suggestion
  desc "Web security auditing toolkit"
```

File: Casks/c/caido.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  homepage "https://caido.io/"
+
+  livecheck do
+    url "https://github.com/caido/caido/"
+  end
+
+  depends_on macos: ">= :catalina"
+
+  app "Caido.app"
+  binary "#{appdir}/Caido.app/Contents/Resources/bin/caido-cli"
```
Comment (khipp) [inline Casks/c/caido.rb:21]:
```suggestion
  binary "#{appdir}/Caido.app/Contents/Resources/bin/caido-cli", target: "caido"
```

**Review by khipp — COMMENTED** [review]
Thank you, @i0ntempest! I left a few minor suggestions below.

File: Casks/c/caido.rb (hunk 3 above)
Comment (i0ntempest) [reply Casks/c/caido.rb:21]: The official CLI release file is "caido-cli". I believe it's better to leave it as this for consistency.

**Review by khipp — APPROVED** [review]
Thank you!

**Pull request merged**